egs/hkust/s5/local/nnet3/compare_wer_general.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+# Copyright 2018  Emotech LTD (Author: Xuechen Liu)
```

Comment:
Please add another script local/chain/compare_wer.sh (or just move this and modify it if there are no non-chain nnet3 example scripts).
Make the name 'compare_wer.sh' not compare_wer_general.sh.
The chain compare_wer.sh script will compare the chain and xent objective functions, see other examples.